### ISSUE TITLE
manually make user admin

### ISSUE THREAD

**Issue by skwerlman** [Cockatrice/Cockatrice#2032]
This is more of a question than an issue, but it's not covered in the servatrice faq, so i thought i'd ask anyway.

I need to convert a servatrice account on my private server to admin. How can i do this?

**Comment by ctrlaltca** [comment]
I'm afraid there is no UI for this at the moment. You need to manually execute this sql query:
`update cockatrice_users set admin=1 where name = 'username'`

1 = admin
2 = moderator

**Comment by Daenyth** [comment]
We can probably get in a CLI argument for servatrice for this

**Comment by ctrlaltca** [comment]
Related: #1391 (promote users to moderator)

**Comment by woogerboy21** [comment]
Wait yes there is a UI method.

**Comment by ctrlaltca** [comment]
> Wait yes there is a UI method. If your an administrator you can right click on the user name with the admin tools enabled and its a menu option.

Maybe i'm wrong, but you can only promote users to moderators, but not to administrators.

**Comment by woogerboy21** [comment]
Your correct, I incorrectly recalled the PR change.  It was only for promotion to moderation.

**Comment by skwerlman** [comment]
Thanks for the command, ctrlaltca

+1 for cli flag, as long as it treats the already running server like a daemon (no restart; new process doesn't block)

**Comment by Daenyth** [comment]
> as long as it treats the already running server like a daemon (no restart; new process doesn't block)

Ehhhhh that's a bit more of a pain. It's a reasonable request but harder to implement. The server code isn't really that flexible.

Maybe just a script for the moment then since it's easy enough with raw sql

**Comment by skwerlman** [comment]
fair enough; i'll open a separate issue for daemon-style server configuration.
